spec/views/prayer_requests/edit.html.erb_spec.rb

```diff
@@ -1,20 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe('prayer_requests/edit') do
-  before do
-    @user1 = FactoryBot.create(:user, email: 'user1@gmail.com')
-    @admin = FactoryBot.create(:admin_user, email: 'admin@gmail.com')
-  end
+  let!(:user1) { FactoryBot.create(:user, email: 'user1@gmail.com') }
+  let!(:admin) { FactoryBot.create(:admin_user, email: 'admin@gmail.com') }
```

Comment:
thanks for fixing the `@` issue with rubocop, let's look into re-enabling that rubocop rule now